src/style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p,
+h1,
+img {
+  margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
do not use sizes in blocks, use mix. 
check list 7
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md